packages/istanbul-lib-instrument/src/read-coverage.js

```diff
@@ -14,7 +14,10 @@ export default function readInitialCoverage (code) {
         allowReturnOutsideFunction: true,
         allowSuperOutsideMethod: true,
         sourceType: "script", // I think ?
-        plugins: ["*"]
+        plugins: [
+          "*",
```

Comment:
The parser doesn't have `*` anymore. Some parser plugins conflict with others, so we require an explicit list of parser plugins to enable.